examples/crm/src/companies/CompanyAvatar.tsx

```diff
@@ -15,7 +15,10 @@ const classes = {
     large: `${PREFIX}-large`,
 };
 
-const StyledAvatar = styled(Avatar)({
+const StyledAvatar = styled(Avatar, {
```

Comment:
Same here, let's use sx. Same for all the demos components

Reply:
I've maintained the styled component because of the custom imgProps